test/integration/testExecute.js

```diff
@@ -137,6 +137,25 @@ describe('Execute test', function () {
       done
     );
   });
+
```

Comment:
The provided integration test shows, that for describeOnly === undefined we have 1 row returned (no argument in the function call - line 157).
Do we check what happens when it is actually set to true?

Reply:
My understanding is that setting describeOnly expects only metadata to be returned. The rowset should be empty.